source/vistas/core/graphics/renderable.py

```diff
@@ -43,12 +43,28 @@ def __init__(self):
         self.scale = Vector3([1, 1, 1])
         self.position = Vector3()
         self.rotation = Vector3()
+        self._bounding_box = None
+
+        self.bbox_vao = -1
+        self.bbox_vertex_buffer = -1
+        self.bbox_index_buffer = -1
+
+        # Init the VAO
         self.bounding_box = BoundingBox(0, 0, 0, 0, 0, 0)
 
-    def render(self, camera):
-        pass
+    def __del__(self):
+        if self.bbox_vao != -1:
+            glDeleteVertexArrays(1, [self.bbox_vao])
+            glDeleteBuffers(1, [self.bbox_vertex_buffer])
+            glDeleteBuffers(1, [self.bbox_index_buffer])
 
-    def render_bounding_box(self, color, camera):
+    @property
+    def bounding_box(self):
+        return self._bounding_box
+
+    @bounding_box.setter
+    def bounding_box(self, bounding_box):
+        self._bounding_box = bounding_box
```

Comment:
The purpose for the setter is that whenever the bounding box dimensions change, we need to overwrite the buffer containing the vertex information and re-enable the attribute array location for shader programs. Doing it here instead of within the `render_bounding_box` method means we are only allocating the buffers one time, and then reusing them whenever we bind the `vertex_array_object`.